code/__DEFINES/weapons.dm

```diff
@@ -95,7 +95,7 @@ Fire delay 15 = 40 rounds per minute
 #define WEAPON_CHARGE		list(mode_name="charge mode", mode_desc="Hold down the trigger, and let loose a more powerful shot", mode_type = /datum/firemode/charge, icon="charge")
 
 #define STUNBOLT			list(mode_name="stun", mode_desc="Stun bolt until they're eating the floortiles", projectile_type=/obj/item/projectile/beam/stun, item_modifystate="stun", fire_sound='sound/weapons/energy/Taser.ogg', icon="stun")
-#define LETHAL				list(mode_name="kill", mode_desc="To defeat the Vagabond, shoot it until it dies", projectile_type=/obj/item/projectile/beam, item_modifystate="kill", fire_sound='sound/weapons/energy/Laser.ogg', icon="kill")
+#define LETHAL				list(mode_name="kill", mode_desc="To defeat the Colonist, shoot it until it dies", projectile_type=/obj/item/projectile/beam, item_modifystate="kill", fire_sound='sound/weapons/energy/Laser.ogg', icon="kill")
```

Comment:
```suggestion
#define LETHAL				list(mode_name="kill", mode_desc="To defeat the Vagabond, shoot it until it dies", projectile_type=/obj/item/projectile/beam, item_modifystate="kill", fire_sound='sound/weapons/energy/Laser.ogg', icon="kill")
```
This works better as Vagabond well being an eris name for assitant, is also a traveler, thematically it fits better then colonists tho its your choice to put this back or not

Reply:
Eh, I wanted to make it Sojourn tematic but sure, if it's alright with you
My other option was "outsider" but it would've been too edgy